tests/cross/src/test/scala/tests/hover/HoverScala3TypeSuite.scala

```diff
@@ -81,7 +81,7 @@ class HoverScala3TypeSuite extends BaseHoverSuite {
        |""".stripMargin,
     "",
     compat = Map(
-      "3.1.3-RC1" ->
+      ">=3.1.3-RC1-bin-20220301-fae7c09-NIGHTLY" ->
```

Comment:
Can we say `>3.1.3-RC1` here? (this is not a strong opinion. If you prefer this way, go ahead 👍  )

-  If we write `>3.1.3-RC1`, the nightly releases of `3.1.3-RC1` before `20220301` will fail, but do we test against older Scala3 nightly release?  
- I feel it's too specific
- If we can write `>3.1.3-RC1`,  we can omit `nightlyDate` field from `Version` case class, and keep it simple :)

Reply:
@tanishiking Initially I wanted to use just `>` but then realized that the actual usage of this might be gte only. Some change happens in specific version so you add a new compat case that should be used from this version and next ones. 
I agree that `>=` is more verbose than `>` but I think at least it's more clear.

> If we write >3.1.3-RC1, the nightly releases of 3.1.3-RC1 before 20220301 will fail, but do we test against older Scala3 nightly release?

We have a ci workflow that tests and backpublishes support of new scala3-nightly versions for the latest Metals version.
Being too specific with versions will help to stabilize this workflow as currently it often fails.

Reply:
Thank you for the detailed explanation! 

> We have a ci workflow that tests and backpublishes support of new scala3-nightly versions for the latest Metals version.
> Being too specific with versions will help to stabilize this workflow as currently it often fails.

I'm convinced that being specific with versions would be better now.
Now, it's LGTM from my side.